src/llmcompressor/modifiers/awq/mappings.py

```diff
@@ -132,7 +132,26 @@ class AWQMapping:
     AWQMapping("re:.*up_proj$", ["re:.*down_proj$"]),
 ]
 
+_bloom_mappings = [
+    AWQMapping(
+        "re:.*input_layernorm$",
+        ["re:.*query_key_value$"]
+    ),
+    AWQMapping(
+        "re:.*query_key_value$",
+        ["re:.*dense$"]
+    ),
+    AWQMapping(
+        "re:.*post_attention_layernorm$",
+        ["re:.*dense_h_to_4h$"]
+    ),
+    AWQMapping(
+        "re:.*dense_h_to_4h$",
+        ["re:.*dense_4h_to_h$"]
+    ),
```

Comment:
I think this a bit off with respect to what's in AutoAWQ [here](https://github.com/casper-hansen/AutoAWQ/blob/main/awq/models/bloom.py#L62-L68)
```suggestion
    AWQMapping(
        "re:.*gelu_impl$",
        ["re:.*dense_4h_to_h$"]
    ),
```